Error during making of debian package.

I added pagespeed module into standard nginx-full debian package, but during compile of package i receive this error:

/tmp/ngx_pagespeed/psol/include/third_party/chromium/src/base/logging.h:289: error: expected unqualified-id before numeric constant
/tmp/ngx_pagespeed/psol/include/third_party/chromium/src/base/logging.h:290: error: expected unqualified-id before numeric constant
make[2]: **\* [objs/addon/src/ngx_pagespeed.o] Error 1
make[2]: Leaving directory `/usr/src/nginx/source/nginx-1.2.7/debian/build-full'
make[1]: *** [build] Error 2
make[1]: Leaving directory`/usr/src/nginx/source/nginx-1.2.7/debian/build-full'
make: **\* [build-arch.full] Error 2
dpkg-buildpackage: error: debian/rules build gave error exit status 2

--- a/src/ngx_pagespeed.h
+++ b/src/ngx_pagespeed.h
@@ -19,6 +19,17 @@
 #ifndef NGX_PAGESPEED_H_
 #define NGX_PAGESPEED_H_
 
+// We might be compiled with syslog.h, which #defines LOG_INFO and LOG_WARNING
+// as ints.  But logging.h assumes they're usable as names, within their
+// namespace, so we need to #undef them before including logging.h
+#ifdef LOG_INFO
+#undef LOG_INFO
+#endif
+#ifdef LOG_WARNING
+#undef LOG_WARNING
+#endif
+#include "base/logging.h"
+
 extern "C" {
   #include <ngx_core.h>
   #include <ngx_http.h>

